scripts/openshiftci-presubmit-e2e-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Do not show token in CI log
+set +x
+
+# show commands
+set -x
+
+# Set necessary env variables
+export PATH="$PATH:$(pwd)"
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
```

Comment:
The `KUBECONFIG` variable is not used in the script. Do we need to keep it?

Reply:
Yes, we dont need it. I put it there when I was not able to make DB connection from E2E test, but later port forwarding fixed the problem and it is not needed anymore.